### PR TITLE
docs(select): add basic readme

### LINKED ISSUE #2148: md-select is missing "Read Me" docs
I saw the alpha 11 has been released. However, is there any "Read Me" for md-select ?

### PR DESCRIPTION
**Pull request by kara**
Closes #2148

### CODE REVIEW

**Comment by jelbourn** [comment]
Can you add documentation to the OVERVIEW.md too? That's where the docs on material.angular.io are generated from.

**Comment by kara** [comment]
@jelbourn Updated with overview!

File: src/lib/select/OVERVIEW.md [hunk 1]
```diff
@@ -1,8 +1,41 @@
 `<md-select>` is a form control for selecting a value from a set of options, similar to the native
-`<select>` element.
+`<select>` element. You can read more about selects in the 
+[Material Design spec](https://material.google.com/components/menus.html).
 
-<!-- example(select-overview) -->
```
Comment (jelbourn) [inline src/lib/select/OVERVIEW.md]: This comment needs to stay. This is what embeds the live example on the docs site.

File: src/lib/select/README.md [hunk 2]
```diff
@@ -1,3 +1,68 @@
-## Work in progress!
+# md-select
 
-The select is still a work in progress, so most features have not been implemented. Not ready for use!
\ No newline at end of file
+`<md-select>` is a form control for selecting a value from a set of options, similar to the native
```
Comment (jelbourn) [inline src/lib/select/README.md]: FYI, I plan on wiping out all of the README files soon (and just pointing to the docs site)

File: src/lib/select/OVERVIEW.md (hunk 1 above)
Comment (kara) [inline src/lib/select/OVERVIEW.md]: Oopsies

File: src/lib/select/README.md (hunk 2 above)
Comment (kara) [inline src/lib/select/README.md]: Yeah, submitted before we had that conversation. Want me to wipe now?

Comment (jelbourn) [inline src/lib/select/README.md]: Doesn't matter to me; might as well keep it before we wipe them all.

**Comment by jelbourn** [comment]
LGTM

**Pull request merged**